src/debugger/nodeDebugWrapper.ts

```diff
@@ -135,8 +136,16 @@ Telemetry.init("react-native-debug-adapter", version, true).then(() => {
             args.target || "simulator"
         ];
 
+        if (!isNullOrUndefined(args.logCatArguments)) { // We add the parameter if it's defined (adapter crashes otherwise)
+            args.args = args.args.concat([args.logCatArguments]);

```

Comment:
I mean that `[1,2,3].concat([[4,5,6]]) = [1,2,3, [4,5,6]]`, and since in this case these are arguments being passed via argv I don't think we can pass an array as a single argument.